webrtc.html

```diff
@@ -4940,8 +4940,16 @@ <h2>Methods</h2>
                       <var>track</var>.</p>
                     </li>
                     <li>
-                      <p>Set <var>sender</var>'s <a>[[\AssociatedMediaStreams]]</a>
-                      to <var>streams</var>.</p>
+                      <p>Set <var>sender</var>'s
+                      <a>[[\AssociatedMediaStreamIds]]</a> to an empty set.
+                      </p>
+                    </li>
+                    <li>
+                      <p>For each <var>stream</var> in <var>streams</var>, add
+                      <var>stream.id</var> to
+                      <a>[[\AssociatedMediaStreamIds]]</a> if it's not already
+                      there.
```

Comment:
Since `[[AssociatedMediaStreamIds]]` was just set to an empty set, won't the IDs *always* be "not already there"? Unless there's a duplicate stream I guess.

Reply:
Yes it's to handle duplicate ids. E.g.
```js
pc.addTrack(track, stream, stream);
```
The old prose didn't remove duplicates, which it probably should?